### PR TITLE
Merge master into mobx

### LINKED ISSUE #4430: Merge master into mobx
It is past due to merge master into mobx. There's a few things we'll get:
* The updated story panel
* Lot of merge conflicts

Sounds fun!

### PR DESCRIPTION
**Pull request by tephenavies**
### Merge from master

Fixes #4430. This time without changing LF to CRLF for a bunch of files.

There are some things we should port from master that are not included in this PR. These are outlined in #4465.

### Checklist

-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Comment by tephenavies** [comment]
To be reviewed and merged after June 30th deadlines.

File: lib/ReactViews/StandardUserInterface/standard-user-interface.scss [hunk 1]
```diff
@@ -19,7 +19,7 @@
   }
 
   .top-element {
-    z-index: 99999 !important;
+    z-index: $notification-window-z-index - 10 !important;
```
Comment (soyarsauce) [inline lib/ReactViews/StandardUserInterface/standard-user-interface.scss:22]: Now I see.. will need some consolidation as they are worlds apart
$front-component-z-index is 99
$notification-window-z-index is 99999
Also related: https://github.com/TerriaJS/terriajs/issues/3958

File: lib/ReactViews/ToolPanel.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+function loadDeltaTool({ onError }) {
+  return function(onLoad) {
+    import("./Tools/DeltaTool/DeltaTool.jsx")
+      .then(module => onLoad(module.default))
+      .catch(onError);
+  };
+}
+
+export default ToolPanel;
```
Comment (soyarsauce) [inline lib/ReactViews/ToolPanel.jsx]: Deleting this

File: lib/ReactViews/Tools/DeltaTool/DatePickers.jsx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      : undefined;
+  return { prevDate, nextDate };
+}
+
+function useDatePickerState() {
+  const [isOpen, setIsOpen] = useState(false);
+  return { isOpen, setIsOpen };
+}
+
+export default withTranslation()(DatePickers);
```
Comment (soyarsauce) [inline lib/ReactViews/Tools/DeltaTool/DatePickers.jsx]: Deleting this

File: lib/ReactViews/Tools/DeltaTool/DeltaTool.jsx [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+function dateParamFormat(date) {
+  return dateFormat(date, "yyyy-mm-dd'T'HH:MM:ss.l'Z'", true);
+}
+
+function dateDisplayFormat(date) {
+  return dateFormat(date, "dd-mm-yyyy", true);
+}
+
+export default withTranslation()(DeltaTool);
```
Comment (soyarsauce) [inline lib/ReactViews/Tools/DeltaTool/DeltaTool.jsx]: Deleting this

File: lib/ReactViews/Tools/DeltaTool/LocationPicker.jsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+function cartesianToDegrees(cartesian) {
+  const carto = Ellipsoid.WGS84.cartesianToCartographic(cartesian);
+  return {
+    longitude: CesiumMath.toDegrees(carto.longitude),
+    latitude: CesiumMath.toDegrees(carto.latitude)
+  };
+}
+
+export default LocationPicker;
```
Comment (soyarsauce) [inline lib/ReactViews/Tools/DeltaTool/LocationPicker.jsx]: Deleting this

File: lib/ReactViews/Tools/DeltaTool/delta-tool.scss [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+  position: relative;
+  width: 0;
+  height: 0;
+  margin-right: 5px;
+}
+
+.datePickerPopup {
+  top: 5px;
+  left: 0px;
+}
```
Comment (soyarsauce) [inline lib/ReactViews/Tools/DeltaTool/delta-tool.scss]: Deleting this

File: lib/ReactViews/tool-panel.scss [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+.loading {
+  display: block;
+  width: 100px;
+  margin: auto;
+  margin-top: 7em;
+  color: white;
+  svg {
+    fill: white;
+  }
+}
```
Comment (soyarsauce) [inline lib/ReactViews/tool-panel.scss]: Deleting this

**Comment by soyarsauce** [comment]
Awaiting ci then merging

**Review by soyarsauce — APPROVED** [review]

**Comment by tephenavies** [comment]
Thanks for cleaning up the old files. For some reason I thought of master-style CatalogItems but not all the master delta stuff.

**Pull request merged**